src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -136,7 +150,7 @@ export interface TradeProps extends GasEstimateProps {
 export function Trade({ trade, outputUSDC, impact, gasUseEstimateUSD }: TradeProps) {
   return (
     <>
-      <CaptionRow gap={0.5}>
+      <CaptionRow gap={0.5} shrink={0}>
```

Comment:
Could `flex-shrink: 0` be defined in `CaptionRow` instead of expanding `Row` to include it for a one-off?

Reply:
sure - any reason we don't want to expand `Row`?